pootle/core/checks/checker.py

```diff
@@ -343,5 +343,5 @@ def update_translated(self):
     def update_untranslated(self):
         """Delete QualityChecks for untranslated Units
         """
-        checks_qs = self.checks_qs.exclude(unit__state__gte=OBSOLETE)
+        checks_qs = self.checks_qs.exclude(unit__state__gte=UNTRANSLATED)
```

Comment:
Shouldn't it be `gt` instead of `gte`. Otherwise untranslated units are excluded.